hudi-utilities/src/test/java/org/apache/hudi/utilities/TestHoodieIndexer.java

```diff
@@ -100,6 +101,16 @@ public void init() throws IOException {
     initMetaClient();
   }
 
+  @AfterAll
+  public static void cleanup() {
```

Comment:
The spark context wasn't being closed which could cause other tests to fail that tried to create their own context

Reply:
this test class can actually extend `org.apache.hudi.testutils.SparkClientFunctionalTestHarness` to make use of most of the resources like spark session and context. should just migrate to use it?

Reply:
Yes, I've updated the code